integration-test/app-different-package/main.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"fmt"
+	"github.com/ahmetalpbalkan/govvv/integration-test/app-different-package/mypkg"
+)
+
+func main() {
+	fmt.Printf("Version=%s\n", mypkg.Version)
+	fmt.Printf("BuildDate=%s\n", mypkg.BuildDate)
```

Comment:
you don't need all these. The purpose of the test is to just test a package other than main builds. So just Version= is ok in this file.